docker.go

```diff
@@ -57,6 +66,21 @@ func runDockerPin(cmd *cobra.Command, args []string) error {
 	return ioutil.WriteFile(ifDash(*dockerfile, "/dev/stdout"), n, 0644)
 }
 
+func runDockerResolve(cmd *cobra.Command, args []string) error {
+	baseImageAndVersion := args[0]
+	dockerClient, err := docker.NewClientWithOpts(docker.FromEnv)
+	if err != nil {
+		return err
+	}
+	di, err := dockerClient.DistributionInspect(cmd.Context(), baseImageAndVersion, "")
+	if err != nil {
+		return err
+	}
+	hashedBase := baseImageAndVersion + "@" + string(di.Descriptor.Digest)
+	_, err = fmt.Println(hashedBase)
```

Comment:
Congratulations, you just became the first person ever to check fmt.Println's return value ;)